projects/app/src/global.css

```
@@ -333,6 +333,13 @@
     weight font. This disables that behaviour.
     */
     font-synthesis: none;
+    /*
+    Make webkit (e.g. Safari) render the font the same way as Figma
+    https://forum.figma.com/ask-the-community-7/font-in-browser-seem-bolder-than-in-the-figma-8146#post54600
+    */
+    -webkit-font-smoothing: antialiased;
+    -moz-osx-font-smoothing: grayscale;
+    font-smooth: never;
```

Biome (2.1.2)

[error] Unknown property is not allowed.

See CSS Specifications and browser specific properties for more details.
To resolve this issue, replace the unknown property with a valid CSS property.

(lint/correctness/noUnknownProperty)